topo_processor/imagery/compressors/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .data_compressor_imagery_historic import DataCompressorImageryHistoric
```

Comment:
might be better to call this a `transformer` and the have a compressor as type of a transformer

Reply:
why is that? are we going to have other types of transformer in the future for lidar and aerial imagery?

Reply:
Yes there will be more transformations later, cropping/filtering also a gdal_translate is a transformation, you don't actually have to compress a file to make a COG.

Reply:
yup will change that